docs/Configuration.md

```diff
@@ -32,6 +32,54 @@ SQLite            | `SQLite`
 
 TBD
 
+## LANGUAGE section
+
+The LANGUAGE section has one key, `lang`.
+
+The value of the `lang` key is a space separated list of
+language tags where each tag must match the regular expression
+`/^[a-z]{2}_[A-Z]{2}$/`.
+
+The two first characters of a language tag are expected to be an
+[ISO 639-1] two-character language code and the two last characters
+are expected to be an [ISO 3166-1 alpha-2] two-character country code.
+A language tag is a locale setting for the available translation
+of messages without ".UTF-8", which is assumed.
+
+Adding a new language to the configuration requires that the
+equivalent MO file is added to Zonemaster-Engine at the correct
+place so that gettext get retrieve it. Removing a language from
+the configuration file just blocks that language from being
+allowed.
```

Comment:
I think this paragraph should be split in two. First we should clarify that having a language tag in the configuration without providing the corresponding translation files is an error. Second we should link to the documentation for adding a new translation.

Reply:
We should come back to this after zonemaster/zonemaster-engine#594 is completed.